config.yaml

```
@@ -1,6 +1,6 @@
 options:
   channel:
     type: string
-    default: "1.29/stable"
+    default: "1.29/beta"
```

I'm not sure why they charm said `1.29/stable`.  That should only be present on the `1.29_release` branch...

I'd rather the default here be something that never changes on the main branch of this charm

```suggestion
    default: "edge"
```

---

charm code in `main` branch will be published to the `latest/edge` channel for this charm. 
charm code in `release_1.29` branch will be published eventually to `1.29/stable`